scripts/install.js

```diff
@@ -55,6 +55,13 @@ console.log('Installing & activating plugins ...');
 composer('update', config.paths.container.app);
 installPluginsDependencies(config);
 
+/**
+ * Install development requirements
+ */
+console.log('Installing development requirements ...');
+run('cp .wp-env.override.json.dist .wp-env.override.json');
+run('npx wp-env run cli wp plugin install --activate query-monitor');
```

Comment:
this will be activated in the current database, so we should execute it after we switch database (`useDatabase` l.83)

Reply:
Good point. I moved it a bit further down.